quickrun.el

```diff
@@ -775,7 +775,12 @@ if you set your own language configuration.")
         (unless (file-directory-p formatted)
           (throw 'quickrun
                  (format "'%s' is not existed directory" it)))
-        (setq quickrun-option-default-directory formatted)))))
+        (let* ((has-space (string-match " " formatted))
+               (quoted-name (shell-quote-argument
+                              (if has-space
+                                (concat "'" formatted "'")
```

Comment:
You might want to change this to double quote? I think path can contain single quote but not the double quote hence should be better in this case.

Reply:
I checked and paths can contain both single and double quotes. My concern with double quotes is that it might try to expand shell variables (if path name contains `$`). Not sure what the best choice is here

Reply:
Oh, cool. What OS do you use? On Windows, it refuses for me to add a double quote. I think `shell-quote-argument` will resolve the expansion from the shell variable. I mean that is what it suppose to do? 😕

Reply:
Ah ok, didn't know that. I use Ubuntu, so generally for linux the only characters not allowed are the `/` and `\0` characters. But I think those will only be seen in extremely rare cases, and `shell-quote-argument` should handle it fine. I'll make the change now

Reply:
Anyway, I have tested using double quote with `shell-quote-argument`. I can ensure you this should work! 😄 All good!